src/services/messengerSync/consumer.js

```diff
@@ -16,17 +16,17 @@ const validateMessage = (content) => {
 const handleMessage = async (incomingMessage) => {
 	const content = JSON.parse(incomingMessage.content.toString());
 	if (!validateMessage) {
-		logger.warning(`invalid message in queue ${QUEUE_INTERNAL}`, incomingMessage);
-		channel.nack(incomingMessage);
+		logger.warning(`MESSENGER SYNC: invalid message in queue ${QUEUE_INTERNAL}`, incomingMessage);
+		channel.reject(incomingMessage, false); // message is invalid an can not be retried
 	}
 	try {
 		const addUserMessageObject = await buildAddUserMessage(content);
 		const outgoingMessage = JSON.stringify(addUserMessageObject);
 		channel.sendToQueue(QUEUE_EXTERNAL, Buffer.from(outgoingMessage), { persistent: true });
 		channel.ack(incomingMessage);
 	} catch (err) {
-		logger.error(`error while handling message in queue ${QUEUE_INTERNAL}`, err);
-		channel.nack(incomingMessage);
+		logger.error(`MESSENGER SYNC: error while handling message in queue ${QUEUE_INTERNAL} `, err);
+		channel.reject(incomingMessage, !incomingMessage.fields.redelivered); // retry message once (the second time it is redelivered)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 135. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5321126)